packages/core/ui/App.js

```diff
@@ -36,28 +33,17 @@ const useStyles = makeStyles({
   },
 })
 
-function App({ contentRect, measureRef, session }) {
-  const theme = useTheme()
-  const margin = theme.spacing(1)
+function App({ session }) {
   const classes = useStyles()
   const { pluginManager } = session
-  const { width } = contentRect.bounds
-  useEffect(() => {
-    if (width) {
-      if (isAlive(session)) {
-        session.updateWidth(width, margin * 2)
-      }
-    }
-  }, [session, width, margin])
 
-  const { visibleDrawerWidget, appWidth, drawerWidth } = session
+  const { visibleDrawerWidget, drawerWidth } = session
 
   return (
     <div
-      ref={measureRef}
       className={classes.root}
       style={{
-        gridTemplateColumns: `[main] ${appWidth}px${
+        gridTemplateColumns: `[main] 1fr${
```

Comment:
can you elaborate on the usage of the fr unit, just something I haven't used before

Reply:
It's the fraction of the available space. Here it just means "the rest of the space available."

https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Basic_Concepts_of_Grid_Layout#The_fr_Unit